databricks/koalas/frame.py

```diff
@@ -314,8 +314,47 @@ def isnull(self):
 
     isna = isnull
 
-    @derived_from(pd.DataFrame)
     def notnull(self):
+        """
+        Detects non-missing values for items in the current Dataframe.
+
+        This function takes a dataframe and indicates whether it's
+        values are valid (not missing, which is ``NaN`` in numeric
+        datatypes, ``None`` or ``NaN`` in objects and ``NaT`` in datetimelike).
+
+
+        See Also
+        --------
+        Dataframe.isnull
+
+        Examples
+        --------
+        >>> df = ks.DataFrame([(.2, .3), (.0, None), (.6, None), (.2, .1)])
+        ...                    (.2, .1)])
+        >>> df.notnull()
+              0      1
+        0  True   True
+        1  True  False
+        2  True  False
+        3  True   True
+
+        >>> pdf = pd.DataFrame([['ant', 'bee', 'cat'], ['dog', NaN, 'fly']])
```

Comment:
change this to
```
df = ks.DataFrame...
```

and then the next line should just be "df"

Reply:
Changed as per requirements.